.travis.yml

```diff
@@ -8,8 +8,9 @@ osx_image: xcode6.4
 
 env:
   matrix:
-    
+
     - CONDA_PY=27
+    - CONDA_PY=34
```

Comment:
@ocefpaf sorry about that.  I saw at least one other package (xarray) that has 2.7 and 3.4-3.6 channels all on conda-forge, so I thought it was OK.  Are they a special case?